### PR TITLE
Fix hastebin command

### LINKED ISSUE #31: hastebin Command Bug
The Hastebin command does also include the prefix when posting to hastebin

E.g `xd hastebin hi` turns into `xd hastebin hi` on hastebin instead of just hi

### PR DESCRIPTION
**Pull request by NyCodeGHG**
Closes #31

### CODE REVIEW

**Review by DRSchlaubi — CHANGES_REQUESTED** [review]
The message was actually intended because it allows you to attach a file with text exceeding 1024 chars

**Comment by NyCodeGHG** [comment]
nvm

**Comment by NyCodeGHG** [comment]
@DRSchlaubi does the argument have to be optional so that you can use text attachments with it?

**Comment by DRSchlaubi** [comment]
Yeah, we could check whether we have an attachment and the input is null and then send an error

File: src/main/kotlin/de/nycode/bankobot/listeners/HastebinUploadListener.kt [hunk 1]
```diff
@@ -50,10 +51,18 @@ const val HASTEBIN_COMMAND = "hastebin"
...
 @ModuleName(GeneralModule)
 internal fun hastebinCommand() = command(HASTEBIN_COMMAND) {
-    invoke(StringArgument.asSlashArgument("Der hochzuladende String")) {
+    invoke(StringArgument.optional().asSlashArgument("Der hochzuladende String")) {
         val hasteContent = message.content
             .substring(message.content.indexOf(HASTEBIN_COMMAND) + HASTEBIN_COMMAND.length, message.content.length)
-        respond(HastebinUtil.postToHastebin(hasteContent))
+
+        when (hasteContent.trim()) {
+            "" -> {
```
Comment (DRSchlaubi) [inline src/main/kotlin/de/nycode/bankobot/listeners/HastebinUploadListener.kt:59]: `String.isBlank()` :D

**Review by DRSchlaubi — CHANGES_REQUESTED** [review]

File: src/main/kotlin/de/nycode/bankobot/listeners/HastebinUploadListener.kt (hunk 1 above)
Comment (DRSchlaubi) [reply src/main/kotlin/de/nycode/bankobot/listeners/HastebinUploadListener.kt:59]: Or is guess there also is isNotBlank()

**Review by DRSchlaubi — APPROVED** [review]
LGTM

File: src/main/kotlin/de/nycode/bankobot/listeners/HastebinUploadListener.kt [hunk 2]
```diff
@@ -51,7 +52,20 @@ const val HASTEBIN_COMMAND = "hastebin"
...
 @ModuleName(GeneralModule)
 internal fun hastebinCommand() = command(HASTEBIN_COMMAND) {
-    invoke(StringArgument.optional().asSlashArgument("Der hochzuladende String")) {
+    invoke(StringArgument.optional().asSlashArgument("Der hochzuladende String")) { text ->
+        if (text == null && message.attachments.any { !it.isImage }) {
+            respondEmbed(
+                Embeds.error(
+                    "Kein Inhalt gefunden!",
+                    "Ich konnte keinen Inhalt finden" +
+                            ", den ich auf hastebin posten könnte!" +
```
Comment (warriorzz) [inline src/main/kotlin/de/nycode/bankobot/listeners/HastebinUploadListener.kt:61]: hm...

**Review by warriorzz — APPROVED** [review]

**Comment by NyCodeGHG** [comment]
wait

**Comment by warriorzz** [comment]
NotLikeThis finally

**Comment by NyCodeGHG** [comment]
@warriorzz are you happy now? :)

**Comment by warriorzz** [comment]
sure

**Pull request merged**